apps/indexer-coordinator/src/yargs.ts

```diff
@@ -82,7 +82,7 @@ function getYargsOption() {
     mmrPath: {
       type: 'string',
       describe: 'The local path to store the mmr data',
-      default: '/home/indexer-service',
+      default: '',
```

Comment:
Better make this param as required other than provide a empty string as default

Reply:
This pr will make sure `mmrPath` to be passed into template.yml. For previously created containers, the `mmrPath` is empty. So the poi path will be `/poi/xxx`. After this pr, with default value `/home/indexer-service`, the poi path might become `/home/indexer-service/poi/xxx`. So I think we need to remove the default values for `mmrPath` in case the newly created containers cannot find the former poi file.

Reply:
I think there have problem to use `/poi/xxxx` without `root path` like `/home`. There will have error to find `/poi/....`

Reply:
OK, updated `mmrPath` with default value `/home`.